xcat-inventory/xcclient/inventory/manager.py

```diff
@@ -478,7 +482,7 @@ def getgitinfo(location):
 
 
 
-def importfromfile(objtypelist,objlist,filepath,dryrun=None,version=None,update=True,dbsession=None,envs=None,rootdir=None):
+def importfromfile(objtypelist,objlist,filepath,dryrun=None,version=None,update=True,dbsession=None,envs=None,rootdir=None,exclude=None):
```

Comment:
It seems you didn't exclude anything in this function, right?

Reply:
fixed